contracts/examples/SimpleInterestTermsContract.sol

```diff
@@ -57,6 +57,12 @@ contract SimpleInterestTermsContract is TermsContract {
       _;
     }
 
+    modifier onlyThisContract(bytes32 agreementId) {
```

Comment:
I support making this a modifier, but wonder if there's a better function name we can choose.  Any ideas?

Reply:
lol. I need to get better at coming up with names for modifiers. Are there any conventions you know of? No better name immediately comes to mind...